tests/mock_crust/mod.rs

```diff
@@ -104,13 +97,16 @@ fn candidate_timeout_resource_proof() {
     poll_and_resend(&mut nodes, &mut []);
 
     assert_eq!(
-        nodes_in_authority(&nodes, &proving_node.name()),
+        nodes.iter().map(TestNode::name).collect_vec(),
         nodes_with_candidate(&nodes),
         "All members of destination section accepted node as candidate"
     );
 
     // Continue after candidate time out:
-    FakeClock::advance_time(60 * 60 * 1000);
+    FakeClock::advance_time(
+        1000 * (routing::test_consts::RESOURCE_PROOF_DURATION_SECS
```

Comment:
Minor: I'd bring `test_consts` into scope to not have to prefix it with `routing::` all the time.